pkg/printer/printer.go

```diff
@@ -14,7 +14,7 @@ var printers = map[string]func(string) (Printer, error){
 }
 
 type Printer interface {
-	Print([]judge.Result) error
+	Print([]judge.Result, bool) error
```

Comment:
I'm wondering if it would make sense to pass an object than contains optional flags, rather than just on unclear `bool` here

Reply:
Yeah agree. Let' me check the effort on changing this.

Reply:
done at https://github.com/doitintl/kube-no-trouble/pull/637/commits/e7e6812e916c784c3283407dab3690b7d2c3d129